data/records.json

```
@@ -125,4 +125,25 @@
         "license": "freeware",
         "upload_date": "12/11/2021"
     }
+    {
+        "name": "SpotiBot - Spotify Automation",
+        "description_short": "The bot makes automatic login in spotify, mutes ads and tries to skip them",
+        "description": "SpotiBot is a automation for muting and skipping ads. You can use it as template to try to create your own spotify automation. Makes automatically login before starts to muting",
+        "tags": ["Spotify", "automation","WebBot", "web","bot"],
+        "programming_language": "python",
+        "technology": ["botcity web framework"],
+        "OS": ["windows"],
+        "thumbnail_url": "https://imgur.com/a/Iys0UG4",
+        "youtube_video": "https://youtu.be/YF3BQoyegN0",

```

This needs to be the embedded URL:
```suggestion
        "youtube_video": "https://www.youtube.com/embed/YF3BQoyegN0",
```

* Could you please make sure that your video is listed as public? Otherwise we can't make it display on the Bot page here.